installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -167,13 +167,13 @@
             "SERVICE_HOSTNAME": "{SVC_HOST}"
           },
           "STG": {
```

Comment:
We can remove `stg` since we don't use it at all.

Reply:
Done